capa/rules.py

```diff
@@ -195,8 +195,14 @@ def parse_feature(key):
         return capa.features.Bytes
     elif key == "number":
         return capa.features.insn.Number
+    elif key.startswith("number/"):
+        arch = key.partition("/")[2]
+        return lambda *args, **kwargs: capa.features.insn.Number(*args, arch=arch, **kwargs)
```

Comment:
added some doc inline:

![image](https://user-images.githubusercontent.com/156560/89317065-1f4feb80-d63a-11ea-8a07-45d14beef323.png)

Reply:
upon further reflection, this is a perfect application of [`functools.partial`](https://docs.python.org/2/library/functools.html#functools.partial) though for some reason i didn't think it supported kwargs on py2.7 (it does). going to switch over to that because it makes the intent more clear.

Reply:
![image](https://user-images.githubusercontent.com/156560/89317487-a309d800-d63a-11ea-832d-b3768287428f.png)

Reply:
great, thanks for the explanation and the improvement!